### ISSUE TITLE
Unspecified error occurs in ie8 related to jquery.js when I switch state

### ISSUE THREAD

**Issue by beninfobelt** [angular-ui/ui-router#481]
Hi, I have come across an error in ie8 that I have pinpointed down to the use of a particular javascript library called nice-scroller:

github.com/inuyaksa/jquery.nicescroll.git

The error does not appear when I load a particular url path:

http://localhost:9000/#/home/projects

However, when I travel away from this url to another one, this error occurs only in ie8 (no issues in latest versions of Chrome and FF):

Unspecified error.
jquery.js
Code:0
URI: http://192.168.1.8:9000/bower_components/jquery/jquery.js

I am sure that I have pinpointed it down to nice-scroller because when I comment out this block in nice_scroller_directive.js, the error does not occur while using ie8:

/*
$elem.niceScroll({
    cursorborder:"#ddd",  
    cursorcolor:"#ddd",  
        cursoropacitymin:"0.0",  
    //cursoropacitymax:0.0,  
        cursorwidth:"10",  
        cursorborderradius:"0px",  
    //scrollspeed:100,  
        touchbehavior:"true",  
        bouncescroll:"true",  
        cursordragontouch:"true"
});
*/

To provide some further background, I use nicescroller in a directive called niceScrollbar and use that directive as an attribute in projects.html:

<div class="canvasMenuControls canvasMenuCtrlPM" nice-scrollbar>

I am curious if anyone has come across any issues with jquery when traveling across different states created with ui-router?

Is there some sort of requirement to use .remove() or .detach() or .off() before the state switches?

I found this stackoverflow link that says "Unspecified errors seem to occur when something (usually a value) isn't set or initialized correctly that the browser is attempting to use." 

http://stackoverflow.com/questions/7633722/what-does-unspecified-error-mean-in-ie

Perhaps, the issue arises because the niceScrollbar directive is not used within templateUrl's associated with the other states (that I click to from /#/home/projects).

I have created a plnkr here but it does not run because I have only included the relevant files. If there are additional files I can add to the plnkr to further clarify/detail my issues, please let me know:

http://plnkr.co/edit/gMaqNoiHN6DfBaJjHe82?p=preview

I am very grateful for any assistance you could provide.

All the best,

Ben
